system-parachains/asset-hubs/asset-hub-polkadot/tests/snowbridge.rs

```diff
@@ -57,19 +59,29 @@ fn network_export_table_works() {
 				None,
 			),
 			// From Ethereum with the Sepolia chain ID instead of Mainnet, not matched.
-			(NetworkId::Ethereum { chain_id: 11155111 }, Here, None),
+			(Ethereum { chain_id: 11155111 }, Here, None),
 		];
 
 		for (network, remote_location, expected_result) in test_data {
 			assert_eq!(
-				NetworkExportTable::<BridgeTable>::exporter_for(
+				NetworkExportTable::<EthereumBridgeTableV1>::exporter_for(
```

Comment:
@claravanstaden just a nit, shouldn't we better test final types EthereumNetworkExportTableV1/EthereumNetworkExportTableV2 here?
```suggestion
				EthereumNetworkExportTableV1::exporter_for(
```

Reply:
Addressed in https://github.com/polkadot-fellows/runtimes/pull/796/commits/62143df7281b6cd9c4e61e692796059f05364191.